packages/headless/src/controllers/commerce/context/cart/headless-cart.ts

```diff
@@ -90,7 +84,7 @@ export interface Cart extends Controller {
    *
    * @param item - The cart item to create, update, or delete.
    */
-  updateItem(item: CartItem): void;
+  updateItemQuantity(item: CartItem): void;
```

Comment:
Why rename this method?

Reply:
We are using both productId and name as keys. It makes it clearer that its intended for quantity updates. Name updates don't behave as expected after this refactor